### DISCUSSION TITLE
We want to have multiple Grouptags

### DISCUSSION THREAD

**Discussion by isolutionspsi** [okieselbach/Autopilot-Manager#5] [Q&A] [answered]
Hi Oliver,

We want to have multiple Grouptags for the user to choice.
We set "useApprovalModeGroupTags" to true.
We tried the value approvalModeGroupTags in the following ways:
**Grouptag1,Grouptag2,
AutoPilotManager,TEST=Grouptag1**

But it didn't work. Can you help us out?

**Comment by okieselbach** [comment, marked as answer]
Hi,

sure, the GroupTag is supported in 2 ways. 

First you have a fixed option: AppConfig:AutopilotManagerConfig:GroupTag. When configured it will just use this Tag. Here is no support for multiple ones and it is also not an drop down. Just the fact that every device imported by AutopilotManager will get this additional tag.

Second option is AppConfig:AutopilotManagerConfig:UseApprovalModeGroupTags = true and AppConfig:AutopilotManagerConfig:ApprovalModeGroupTags which is used **only** when the Autopilot Manager is running in AppConfig:AutopilotManagerConfig:ApprovalMode = true.
So, only in the Approval Mode (Helpdesk mode) you get the drop down with the various Group tags.
The reason behind this is that typical end users are not familiar with the concept and giving them the option to decide here something is imho too much and finally resulting in wrong selections and this might end up in wrong assignments etc. More effort to fix in the end than to bring any value. End users are not getting any dropdown for group tags therefore. In ApprovalMode I have Helpdesk engineers taking care of the import and they typically know more from IT and can decide what to pick. So, they get the chance of some predefined values. Also I decided for predefined values to prevent spelling issues if it would be just a text field, which would result again in wrong assignments and additional work to fix all this.

best,
Oliver

**Reply by kaiwessels** [reply]
Hi Oliver,
Thanks for your post. We wants 2 Options with a Dropdown. 

AppConfig:AutopilotManagerConfig:ApprovalMode > true
AppConfig:AutopilotManagerConfig:UseApprovalUserAsAssignedUser > false
AppConfig:AutopilotManagerConfig:GroupTag > AutopilotManager

Do you can give us a Example of 2 GroupTags. Because i have tried a few constelations without success.
AppConfig:AutopilotManagerConfig:ApprovalModeGroupTags > AutopilotManager,NONE=No Group Tag
AppConfig:AutopilotManagerConfig:ApprovalModeGroupTags > AutopilotManager,EndUser=EndUser,Engineer=Engineer

**Reply by okieselbach** [reply]
Hi,

sure. just configure it like this:

AppConfig:AutopilotManagerConfig:ApprovalMode > true
AppConfig:AutopilotManagerConfig:UseApprovalUserAsAssignedUser > false
AppConfig:AutopilotManagerConfig:ApprovalModeGroupTags > EndUser,Engineer

This will result in a dropdown with EndUser and Engineer

best,
Oliver

**Reply by kaiwessels** [reply]
Thanks for your quick response. I have configured the 3 Options but does not receive a dropdown. To be sure the settings are refreshed i have restartet the service. Is there something else what can cause the dropdown not to be displayed ?

BR Kai

**Reply by kaiwessels** [reply]
Found the Problem,

i have set the AppConfig:AutopilotManagerConfig:UseApprovalModeGroupTags > true. And afterwards i have received the Dropdown.

BR Kai
